### ISSUE TITLE
`enter: false` support

### ISSUE THREAD

**Issue by tony** [tmux-python/tmuxp#53]
~Blocked by #751~

Just to leave the console text in the shell.

Such as creating a new pane, running commands in `shell_command_before` then leaving `vim file.py` in the cli window, but not actually sending the carriage return to start it.

Duplicate: #293 #735

**Comment by thiagowfx** [comment]
Hi, was this ever implemented? If yes, a pointer to the docs would be much appreciated. Thank you!

**Comment by thiagowfx** [comment]
Duplicate of https://github.com/tmux-python/tmuxp/issues/293

**Comment by tony** [comment]
@thiagowfx 

Live in 1.10.0b1 [pypi](https://pypi.org/project/tmuxp/1.10.0b1/) [branch](https://github.com/tmux-python/tmuxp/tree/v1.10.0b1)

`pip install --user --upgrade tmuxp==1.10.0b1`

or `pipx install --suffix=@1.10.0b1 'tmuxp==1.10.0b1'`  then `tmuxp@1.10.0b1 load <yaml file>`

docs: [skip command execution](https://tmuxp.git-pull.com/examples.html#skip-command-execution)

```yaml
session_name: Should not execute
windows:
- panes:
  - shell_command: echo "___$((1 + 3))___"
    enter: false
```

**Comment by tony** [comment]
I believe this support may be flawed. It's per-pane, not per-command.

```yaml
session_name: Should not execute
windows:
- panes:
  - shell_command: echo "___$((1 + 3))___"
    enter: false
```

This is equivalent to:

```yaml
session_name: Should not execute
windows:
- panes:
  - shell_command:
    - echo "___$((1 + 3))___"
    enter: false
```

We don't per-command options yet, e.g. this syntax doesn't work:

```yaml
session_name: Should not execute
windows:
- panes:
  - shell_command:
    - my first pane command
    - cmd: my second pane command
    - cmd: echo "___$((1 + 3))___"
      enter: false
```

**Comment by tony** [comment]
@thiagowfx 

Live in 1.10.0b3 [pypi](https://pypi.org/project/tmuxp/1.10.0b3/) [branch](https://github.com/tmux-python/tmuxp/tree/v1.10.0b3)

`pip install --user --upgrade tmuxp==1.10.0b3`

or `pipx install --suffix=@1.10.0b1 'tmuxp==1.10.0b3'`  then `tmuxp@1.10.0b3 load <yaml file>`

docs: [skip command execution](https://tmuxp.git-pull.com/examples.html#skip-command-execution)

```yaml
session_name: Should not execute
windows:
- panes:
  - shell_command:
    - echo "normal command" 
    - cmd: echo "___$((1 + 3))___"
      enter: false
```

**Comment by thiagowfx** [comment]
Thanks for your work, Tony. I’m excited to try this out.

**Comment by tony** [comment]
> Thanks for your work, Tony. I’m excited to try this out.

yes if you'd like to try the Prerelease (above) let me know how it works. It'd be nice to have a tester or two before this goes live

**Comment by tony** [comment]
Added in v1.10

**Comment by thiagowfx** [comment]
Did some quick testing today (v1.10) and it seems to work fine.

**Comment by tony** [comment]
@thiagowfx Thank you!
